src/libre/ast.c

```diff
@@ -428,7 +492,6 @@ ast_make_expr_alt(enum re_flags re_flags)
 
 	res->u.alt.n = malloc(res->u.alt.alloc * sizeof *res->u.alt.n);
```

Comment:
Prefer `calloc` or check for integer overflow in argument expression.

Reply:
This is probably a situation for `calloc`.